torch/distributed/optim/__init__.py

```diff
@@ -1,6 +1,7 @@
 """
 :mod:`torch.distributed.optim` exposes DistributedOptimizer, which takes a list
 of remote parameters and runs the optimizer locally on the workers where the
```

Comment:
Should "remote parameters" link to RRefs or something telling the user how to create remote parameters?